opensrp-reveal/src/main/java/org/smartregister/reveal/util/GeoJsonUtils.java

```diff
@@ -42,7 +45,7 @@
 public class GeoJsonUtils {
 
 
-    public static String getGeoJsonFromStructuresAndTasks(List<Location> structures, Map<String, Set<Task>> tasks, String indexCase) {
+    public static String getGeoJsonFromStructuresAndTasks(List<Location> structures, Map<String, Set<Task>> tasks, String indexCase, Map<String, StructureDetails> structureNames) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'getGeoJsonFromStructuresAndTasks(List, Map, String, Map)' has an NPath complexity of 2993](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4684568)